lib/file_picker.dart

```diff
@@ -3,3 +3,9 @@ library file_picker;
 export './src/file_picker.dart';
 export './src/platform_file.dart';
 export './src/file_picker_result.dart';
+export './src/file_picker_macos.dart';
+export './src/linux/file_picker_linux.dart';
+export './src/file_picker_io.dart';
+// Conditional export needed for web to successfully compile.
+export './src/windows/file_picker_windows_stub.dart'
+    if (dart.library.ffi) './src/windows/file_picker_windows_real.dart';
```

Comment:
Can we keep the `file_picker_windows.dart` filename for the real implementation? (rather than `file_picker_windows_real.dart`)
The fact that the fake one is called `file_picker_windows_stub.dart` clarifies its use in my opinion.

Reply:
Have no strong opinion about this. So yes, done :)